src/service-catalog.ts

```diff
@@ -166,4 +166,44 @@ export async function describeProvisioningParameters(productId: string, provisio
   });
 
   return provisioningParameters;
+}
+
+/**
+ * Makes a series of calls to service catalog to get all information
+ * required to provision a service catalog product.
+ *
+ * @param options
+ * @returns ProductVersionData aggregate of provisioning artifact details.
+ */
+export async function describeProvisionedProduct( options: DescribeProvisionedProductOptions = {}): Promise<ProductDataAggregate> {
```

Comment:
this should be renamed, either describeProduct or Product version

ProvisionedProduct not really accurate